core/primitives/src/errors.rs

```diff
@@ -624,17 +624,7 @@ impl Display for InvalidAccessKeyError {
 impl std::error::Error for InvalidAccessKeyError {}
 
 /// Happens when the input balance doesn't match the output balance in Runtime apply.
-#[derive(
-    BorshSerialize,
-    BorshDeserialize,
-    Debug,
-    Clone,
-    PartialEq,
-    Eq,
-    RpcError,
-    serde::Deserialize,
-    serde::Serialize,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, RpcError, serde::Deserialize, serde::Serialize)]
```

Comment:
I removed `BorshSerialize` and `BorshDeserialize` because it's unnecessary. Not having it makes it obvious that we are not relying on the brosh serialized format, hence we can add fields without the world breaking down.